File: week04-2.py
# week04-2.py是今天的 LeetCode 挑戰題 easy題(二合一)
# LeetCode 2523. Maximum Count of Positive Integer and Negative Integer
# 找出"有幾個正數" "有幾個負數"，大的那個數量回傳
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = 0
        neg = 0
        for i in range(len(nums)):#迴圈裡更新他
            if nums[i]>0:pos += 1
            if nums[i]<0:neg += 1
        return max(pos, neg)

File: week04-2.c
int maximumCount(int* nums, int numsSize) {
   int pos = 0, neg = 0; // 癹伴玡,常琌0

   for(int i=0;i<numsSize;i++){
        if(nums[i]>0) pos++; // タ计
        if(nums[i]<0) neg++; // 璽计
    }// 癹伴い丁穝 э

    //癹伴рウㄓノ
    if(pos>neg) return pos; //タ计ゑ耕タ计
    else return neg;//ぃ礛碞璽计
}
